src/Components/Link/component.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+
+type LinkProps = {
+    label: string,
+    linkTo: string
```

Comment:
All props that all not used in this exact file are not need to be mentioned here.